### DISCUSSION TITLE
Can't click annotation in AR mode

### DISCUSSION THREAD

**Discussion by doubleespresso1** [google/model-viewer#3699] [Q&A]
### Description

Hello, annotation aren't clickable in AR mode. When i'm trying to click one of them my model just moving instead of camera navigation.
#### Live Demo
https://modelviewer.dev/examples/annotations/index.html#cameraViews

### Version
On offical model-viewer website
-->
- model-viewer: vX.X.X

### Browser Affected

- [ x] Chrome, version: 103.0.5060.134
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [x ] WebXR
- [x ] SceneViewer
- [ x] QuickLook

**Comment by elalish** [comment]
Well, this is expected behavior, but I see what you mean. The whole point of AR mode is that the camera is controlled by the user's phone position, rather than by our APIs. What UX would you expect or desire in this case? For instance, say the hotspot is on top, with the camera looking down, but you're in AR mode looking horizontally? I think respecting that click listener would look pretty strange...
